docs/fsharp/language-reference/literals.md

```diff
@@ -48,7 +48,21 @@ let x = "a" + "b" // evaluated at run-time
 let y = "a" + "b" // evaluated at compile-time
 ```
 
-For example, this distinction matters when calling an [external function](functions/external-functions.md), because `DllImport` is an attribute that needs to know the value of `myDLL` during compilation. Without the `[<Literal>]` declaration, this code would fail to compile:
+> [!NOTE]
+> Functions cannot be used to compute [<Literal>] values because literals must be determined at compile-time and cannot depend on runtime evaluation.
```

Comment:
Code fence "literal"

```suggestion
> Functions cannot be used to compute `[<Literal>]` values because literals must be determined at compile-time and cannot depend on runtime evaluation.
```